include/openthread/message.h

```diff
@@ -295,6 +307,18 @@ uint16_t otMessageRead(const otMessage *aMessage, uint16_t aOffset, void *aBuf,
  */
 int otMessageWrite(otMessage *aMessage, uint16_t aOffset, const void *aBuf, uint16_t aLength);
 
+/**
+ * This function registers a callback on message. The callback is invoked to indicate the final status of message
+ * transmission.
+ *
+ * A sub-sequent call to this function will overwrite the previous callback handler.
```

Comment:
is the spelling `sub-sequent` intentional ?

Reply:
Not really. I changed it.